src/IK/include/BiomechanicalAnalysis/IK/InverseKinematics.h

```diff
@@ -127,6 +127,9 @@ class HumanIK
 
     Eigen::VectorXd m_calibrationJointPositions; /** Joint positions for calibration */
 
+    Eigen::VectorXd jointPositionSetPoint; /** Custom set point for the regularization task */
```

Comment:
```suggestion
    Eigen::VectorXd m_jointPositionSetPoint; /** Custom set point for the regularization task */
```

For this class, a `m_` suffix is added to the object that belongs to the class such that it is easier to recognise them in the code.